### ISSUE TITLE
Sonarqube 9.3 plugin 1.10.1 no create comments with merge request

### ISSUE THREAD

**Issue by franviera92** [mc1arke/sonarqube-community-branch-plugin#565]
Hi all,  i need help with plugin and sonarqube 9.3

> 2022.03.19 20:38:25 INFO  ce[][o.s.c.t.CeWorkerImpl] Execute task | project=sonarqubedemo-34289510 | type=REPORT | pullRequest=27 | id=AX-j5w-rMAZKaCpQ05Tq | submitter=gitlab
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Extract report | status=SUCCESS | time=88ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist scanner context | status=SUCCESS | time=4ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Propagate analysis warnings from scanner report | status=SUCCESS | time=3ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Generate analysis UUID | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load analysis metadata | status=SUCCESS | time=4ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Initialize | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Build tree of components | components=133 | status=SUCCESS | time=14ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Validate project | status=SUCCESS | time=4ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load quality profiles | status=SUCCESS | time=170ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load Quality gate | status=SUCCESS | time=5ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load new code period | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Detect file moves | reportFiles=112 | dbFiles=112 | addedFiles=0 | status=SUCCESS | time=5ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load duplications | duplications=6 | status=SUCCESS | time=6ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute cross project duplications | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute size measures | status=SUCCESS | time=11ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute new coverage | status=SUCCESS | time=16ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute coverage measures | status=SUCCESS | time=11ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute comment measures | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute duplication measures | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute size measures on new code | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute language distribution | status=SUCCESS | time=1ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute test measures | status=SUCCESS | time=0ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute complexity measures | status=SUCCESS | time=1ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Load measure computers | status=SUCCESS | time=1ms
2022.03.19 20:38:25 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute Quality Profile status | status=SUCCESS | time=6ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Execute component visitors | status=SUCCESS | time=534ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Checks executed after computation of measures | status=SUCCESS | time=0ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute Quality Gate measures | status=SUCCESS | time=2ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Compute Quality profile measures | status=SUCCESS | time=1ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Generate Quality profile events | status=SUCCESS | time=4ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Generate Quality gate events | status=SUCCESS | time=0ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Check upgrade possibility for not analyzed code files. | status=SUCCESS | time=0ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist components | status=SUCCESS | time=12ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist analysis | status=SUCCESS | time=3ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist analysis properties | status=SUCCESS | time=4ms
2022.03.19 20:38:26 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist measures | inserts=90 | status=SUCCESS | time=9ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist live measures | insertsOrUpdates=7260 | status=SUCCESS | time=881ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist duplication data | insertsOrUpdates=0 | status=SUCCESS | time=4ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist new ad hoc Rules | status=SUCCESS | time=1ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist issues | cacheSize=960 bytes | inserts=0 | updates=2 | merged=0 | status=SUCCESS | time=8ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist project links | status=SUCCESS | time=0ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist events | status=SUCCESS | time=5ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist sources | status=SUCCESS | time=77ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Persist cross project duplications | status=SUCCESS | time=0ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Enable analysis | status=SUCCESS | time=4ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Update last usage date of quality profiles | status=SUCCESS | time=5ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Purge db | status=SUCCESS | time=30ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Index analysis | status=SUCCESS | time=218ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Update need issue sync for branch | status=SUCCESS | time=3ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Send issue notifications | status=SUCCESS | time=0ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Publish task results | status=SUCCESS | time=0ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.s.ComputationStepExecutor] Trigger refresh of Portfolios and Applications | status=SUCCESS | time=0ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Webhooks | globalWebhooks=0 | projectWebhooks=0 | status=SUCCESS | time=3ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.p.a.p.PostProjectAnalysisTasksExecutor] Pull Request Decoration | status=SUCCESS | time=1ms
2022.03.19 20:38:27 INFO  ce[AX-j5w-rMAZKaCpQ05Tq][o.s.c.t.CeWorkerImpl] Executed task | project=sonarqubedemo-34289510 | type=REPORT | pullRequest=27 | id=AX-j5w-rMAZKaCpQ05Tq | submitter=gitlab | status=SUCCESS | time=2283ms

**Comment by franviera92** [comment]
sorry my error

**Comment by LGhoull** [comment]
@franviera92 I have a similar issue. How did you solve this?

**Comment by franviera92** [comment]
> @franviera92 I have a similar issue. How did you solve this?

hello, each project sonar have your general settings,  there should see the option DevOps Platform Integration

**Comment by LGhoull** [comment]
@franviera92 Yes. I have already created a configuration for my Gitlab instance under the GitLab tab. I have entered [instance].com in the configuration name and https://[instance].com/api/v4 in the API URL. I have activated Gitlab Authentication and set the Gitlab URL to https://[instance].com. The error persists. Before I updated sonarqube, i would always see the sonar.mr job (the job i communicate to sonarqube with) and an external "sonar" job. The external sonar job no longer appears.
